.gitignore

```diff
@@ -207,4 +207,7 @@ codealike.json
 
 # Rider
 .idea/
-global.json
\ No newline at end of file
+global.json
```

Comment:
you and your global json

Reply:
I didn't explicity add it so maybe it's a mac/rider thing?